cmd/fpga_admissionwebhook/README.md

```diff
@@ -105,7 +87,14 @@ spec:
     ...
 ```
 
-### Build the webhook image
+## Getting the source code
+
+```bash
+$ mkdir -p $(go env GOPATH)/src/github.com/intel
+$ git clone https://github.com/intel/intel-device-plugins-for-kubernetes $(go env GOPATH)/src/github.com/intel/intel-device-plugins-for-kubernetes
```

Comment:
Do we still need to clone? Can we just use kubectl -k <https://github.com/intel/intel-device-plugins-for-kubernetes/<kustomize path>> for this deployment?

Reply:
Theoretically yes, but we still need sources to build images.

Reply:
We should have images uploaded to docker hub, shouldn't we?

Reply:
Ed's comments was one of the goals we set in the [conversation](https://github.com/intel/intel-device-plugins-for-kubernetes/issues/356#issuecomment-644696747). We'd want to re-word the READMEs so that simple try-out deployments do not ask repo cloning.

Reply:
yes, the images are available in docker hub

Reply:
Ok, updated to deploy right from github. Now the instruction is even shorter.